### PR TITLE
Added pt_PT Readme

### PR DESCRIPTION
**Pull request by qlpqlp**
Added pt_PT Readme

### CODE REVIEW

**Comment by patricklodder** [comment]
Added `help wanted` -> needs a sanity check by a native speaker.

File: doc/intl/pt_PT/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+Este repositório possui 3 principais ramificações (branchs), são estas:
+
+- **master:** Estável, contém a ultima versão da ultima atualização principal *major.minor*.
+- **maintenance:** Estável, contém a ultima versão de atualizações anteriores, que ainda estão em manutenção. Formato: ```<version>-maint```
+- **development:** Instável, contém código novo para atualizações planeadas. Formato: ```<version>-dev```
+
+*As ramificações Master e Maintenance, são exclusivamente mutáveis por lançamento. Atualizações*
+*planeadas terão sempre uma ramificação de desenvolvimento e as solicitações de inclusão deverão ser*
+*encaminhadas por meio destas. Ramificações de manutenção existem apenas para **correção de bugs,***
+*por favor, encaminhem novos recursos na ramificação de desenvolvimento com a versão mais alta.*
```
Comment (vhs94) [inline doc/intl/pt_PT/README.md:64]: not trying to be picky with the `atualização` but just wanted to throw a light on it to keep consistency with line 32 `"actualizadas"`

```suggestion
- **master:** Estável, contém a ultima versão da ultima actualização principal *major.minor*.
- **maintenance:** Estável, contém a ultima versão de actualizações anteriores, que ainda estão em manutenção. Formato: ```<version>-maint```
- **development:** Instável, contém código novo para actualizações planeadas. Formato: ```<version>-dev```

*As ramificações Master e Maintenance, são exclusivamente mutáveis por lançamento. Actualizações*
*planeadas terão sempre uma ramificação de desenvolvimento e as solicitações de inclusão deverão ser*
*encaminhadas por meio destas. Ramificações de manutenção existem apenas para **correção de bugs,***
*por favor, encaminhem novos recursos na ramificação de desenvolvimento com a versão mais alta.*
```

**Review by vhs94 — CHANGES_REQUESTED** [review]
Nice work! 🎉 just added a few small typos fixes and suggestions

File: doc/intl/pt_PT/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+| P2P      |   22556 |   44556 |   18444 |
+| RPC      |   22555 |   44555 |   18332 |
+
+## Desenvolvimento continuo - Plano, a Lua 🌒
+
+O Dogecoin Core é um programa de código aberto gerido pela comunidade. O processo de desenvolvimento é aberto e visivel publicamente; qualquer um pode ver, discutir e trabalhar no programa.
+
+Recursos principais de Desenvolvimento:
+
+* [Projetos do Github](https://github.com/dogecoin/dogecoin/projects) é utilizado para conduzir trabalhos planeado ou que estejam em desenvolvimento para as próximas atualizações.
```
[the remark this answers is not in the document]
Comment (qlpqlp) [reply doc/intl/pt_PT/README.md:47]: In this case, the word "actualizações" do the the new Portuguese spelling agreement its "atualizações" without the "c"

http://www.portaldalinguaportuguesa.org/?action=novoacordo&act=list&version=pe
https://www.dge.mec.pt/acordo-ortografico

Is that OK with you?

[the remark this answers is not in the document]
Comment (vhs94) [reply doc/intl/pt_PT/README.md:47]: @qlpqlp yep! totally fine, but then probably it's better to change like 32 

` e se manter actualizadas` > ` e se manter atualizadas` to keep consistency, right?

File: doc/intl/pt_PT/README.md (hunk 1 above)
Comment (vhs94) [reply doc/intl/pt_PT/README.md:64]: yep! even better actually, good job :)

**Comment by qlpqlp** [comment]
@vhs94 can you review again if everything is OK in your end? :)
Much thanks in advance :)

**Review by vhs94 — APPROVED** [review]

**Comment by vhs94** [comment]
> @vhs94 can you review again if everything is OK in your end? :) Much thanks in advance :)

looking fine from my side, approved 💯  🥇

**Review by patricklodder — APPROVED** [review]
utACK - language review was done by @vhs94

**Pull request merged**